opslevel.yml

```diff
@@ -0,0 +1,13 @@
+---
+version: 1
+repository:
+  name: rules
+  owner: dx_extensibility
+  lifecycle: general_availability
```

Comment:
This will need to be switched to `deprecation` after October 16, 2023

https://oktawiki.atlassian.net/wiki/spaces/A0ENG/pages/2698906834/Service+Catalog+Manifest#Aliases